src/compas_model/model/group_node.py

```diff
@@ -0,0 +1,35 @@
+from compas.datastructures import TreeNode
+from .element_node import ElementNode  # noqa: F401
+
+
+class GroupNode(TreeNode):
+    """Class representing nodes containing elements in an element tree.
+
+    Parameters
+    ----------
+    name : str
+        The name of the group node.
+
+    """
+
+    @property
+    def __data__(self):
+        # type: () -> dict
+        return {
+            "children": [child.__data__ for child in self.children],
+            "name": self.name,
+        }
+
+    @classmethod
+    def __from_data__(cls, data):
+        raise ValueError(
```

Comment:
In ElementTree class it is `Exception`, here is `ValueError`.
Should it be just `Exception`? The same comment is for GroupNode.